src/Lykke.AlgoStore.Api/Controllers/AlgoStoreUsersController.cs

```diff
@@ -50,5 +53,19 @@ public AlgoStoreUsersController(ISecurityClient securityClient)
 
             return Ok(Mapper.Map<AlgoStoreUserDataModel>(result));
         }
+
+        [HttpGet("me/instances")]
```

Comment:
probably we shouldn't use "me" as part rest api endpoint. What about "userInstances"?